cmd/tsidp/tsidp.go

```diff
@@ -764,7 +764,23 @@ var (
 	openIDSupportedSigningAlgos = views.SliceOf([]string{string(jose.RS256)})
 )
 
+func (s *idpServer) addCORSHeaders(w http.ResponseWriter) {
```

Comment:
rename this to `addWildcardCORSHeaders` or inline these in `serveOpenIDConfig`, just in case someone adds this call to other handlers via autocomplete without reading the contents of the function.